### ISSUE TITLE
Suppress the exception of missing dissectors and continue parsing with null values

### ISSUE THREAD

**Issue by vn-techie** [nielsbasjes/logparser#32]
- Can we suppress the missing dissectors instead exception of throwing "nl.basjes.parse.core.exceptions.MissingDissectorsException"

- Null values can be assigned in case of dissectors not found

- This behavior can be controlled overs a configuration to the parser, if needed

**Comment by nielsbasjes** [comment]
I just implemented a simple way to do this in the Java API.
After creating the parser yet before actually parsing the first logline you simply call the method ignoreMissingDissectors() as you can see here:

https://github.com/nielsbasjes/logparser/blob/master/httpdlog/httpdlog-parser/src/test/java/nl/basjes/parse/httpdlog/ApacheHttpdLogParserTest.java#L468

Please have a look and try it this is what you need.

Thanks.

**Comment by vn-techie** [comment]
Thanks for the prompt support. I did look into the code. Looks good. I will verify the same with published maven jar

**Comment by vn-techie** [comment]
Is the 2.7-snapshot publised?

<dependency>
	    <groupId>nl.basjes.parse.httpdlog</groupId>
	    <artifactId>httpdlog-parser</artifactId>
	    <version>2.7-snapshot</version>
	</dependency>

**Comment by vn-techie** [comment]
Could you publish latest httpdlog-parser and parser-core in maven

**Comment by nielsbasjes** [comment]
Yes,  I published the 2.7-SNAPSHOT version (Note: it is "SNAPSHOT" not "snapshot").
See here: https://oss.sonatype.org/content/groups/public/nl/basjes/parse/parser-core/2.7-SNAPSHOT/
To use in your project add this to your pom.xml:

```
<repositories>
  <repository>
    <id>sonatype-oss-public</id>
    <url>https://oss.sonatype.org/content/groups/public/</url>
    <releases>
      <enabled>true</enabled>
    </releases>
    <snapshots>
      <enabled>true</enabled>
    </snapshots>
  </repository>
</repositories>



```

**Comment by nielsbasjes** [comment]
I'm considering this request to be fixed.

**Comment by vn-techie** [comment]
Thanks. Its working as expected. Btw, any time line in mind to publish stable version of 2.7?

**Comment by nielsbasjes** [comment]
I released version 2.7 yesterday.
